<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>571</width>
    <height>470</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget_6" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>571</width>
      <height>471</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>571</width>
      <height>471</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #e2e2a9</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLineEdit" name="lineEdit">
       <property name="maximumSize">
        <size>
         <width>559</width>
         <height>30</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: #c7c7c7;
border-radius: 10px</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string> Search</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="widget" native="true">
       <property name="maximumSize">
        <size>
         <width>559</width>
         <height>425</height>
        </size>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QWidget" name="widget_2" native="true">
          <property name="maximumSize">
           <size>
            <width>547</width>
            <height>413</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">set-coulor:rgb(170, 170, 127)</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QWidget" name="widget_3" native="true">
             <property name="maximumSize">
              <size>
               <width>535</width>
               <height>401</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: #aaaa7f;
border-radius: 5px</string>
             </property>
             <widget class="QStackedWidget" name="stackedWidget">
              <property name="geometry">
               <rect>
                <x>10</x>
                <y>10</y>
                <width>67</width>
                <height>371</height>
               </rect>
              </property>
              <property name="maximumSize">
               <size>
                <width>67</width>
                <height>371</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color: #ffffff
</string>
              </property>
              <property name="currentIndex">
               <number>1</number>
              </property>
              <widget class="QWidget" name="page"/>
              <widget class="QWidget" name="page_2">
               <layout class="QVBoxLayout" name="verticalLayout">
                <item>
                 <widget class="QWidget" name="widget_4" native="true">
                  <property name="maximumSize">
                   <size>
                    <width>16777215</width>
                    <height>16777215</height>
                   </size>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">background-color: #e2e2a9;
border-radius: 5px</string>
                  </property>
                  <widget class="QLabel" name="label_5">
                   <property name="geometry">
                    <rect>
                     <x>7</x>
                     <y>5</y>
                     <width>35</width>
                     <height>35</height>
                    </rect>
                   </property>
                   <property name="maximumSize">
                    <size>
                     <width>35</width>
                     <height>35</height>
                    </size>
                   </property>
                   <property name="text">
                    <string/>
                   </property>
                   <property name="pixmap">
                    <pixmap>../img/food.png</pixmap>
                   </property>
                   <property name="scaledContents">
                    <bool>true</bool>
                   </property>
                  </widget>
                 </widget>
                </item>
                <item>
                 <widget class="QWidget" name="widget_5" native="true">
                  <property name="styleSheet">
                   <string notr="true">background-color: #e2e2a9;
border-radius: 5px</string>
                  </property>
                  <widget class="QLabel" name="label_8">
                   <property name="geometry">
                    <rect>
                     <x>7</x>
                     <y>5</y>
                     <width>35</width>
                     <height>35</height>
                    </rect>
                   </property>
                   <property name="maximumSize">
                    <size>
                     <width>35</width>
                     <height>35</height>
                    </size>
                   </property>
                   <property name="text">
                    <string/>
                   </property>
                   <property name="pixmap">
                    <pixmap>../img/menu.png</pixmap>
                   </property>
                   <property name="scaledContents">
                    <bool>true</bool>
                   </property>
                  </widget>
                 </widget>
                </item>
                <item>
                 <widget class="QWidget" name="widget_8" native="true">
                  <property name="styleSheet">
                   <string notr="true">background-color: #e2e2a9;
border-radius: 5px</string>
                  </property>
                  <widget class="QLabel" name="label_9">
                   <property name="geometry">
                    <rect>
                     <x>7</x>
                     <y>5</y>
                     <width>35</width>
                     <height>35</height>
                    </rect>
                   </property>
                   <property name="maximumSize">
                    <size>
                     <width>35</width>
                     <height>35</height>
                    </size>
                   </property>
                   <property name="text">
                    <string/>
                   </property>
                   <property name="pixmap">
                    <pixmap>../img/home.png</pixmap>
                   </property>
                   <property name="scaledContents">
                    <bool>true</bool>
                   </property>
                  </widget>
                 </widget>
                </item>
                <item>
                 <widget class="QWidget" name="widget_9" native="true">
                  <property name="styleSheet">
                   <string notr="true">background-color: #e2e2a9;
border-radius: 5px</string>
                  </property>
                  <widget class="QLabel" name="label_10">
                   <property name="geometry">
                    <rect>
                     <x>7</x>
                     <y>5</y>
                     <width>35</width>
                     <height>35</height>
                    </rect>
                   </property>
                   <property name="maximumSize">
                    <size>
                     <width>35</width>
                     <height>35</height>
                    </size>
                   </property>
                   <property name="text">
                    <string/>
                   </property>
                   <property name="pixmap">
                    <pixmap>../img/ranking.png</pixmap>
                   </property>
                   <property name="scaledContents">
                    <bool>true</bool>
                   </property>
                  </widget>
                 </widget>
                </item>
                <item>
                 <widget class="QWidget" name="widget_10" native="true">
                  <property name="styleSheet">
                   <string notr="true">background-color: #e2e2a9;
border-radius: 5px</string>
                  </property>
                  <widget class="QLabel" name="label_11">
                   <property name="geometry">
                    <rect>
                     <x>7</x>
                     <y>5</y>
                     <width>35</width>
                     <height>35</height>
                    </rect>
                   </property>
                   <property name="maximumSize">
                    <size>
                     <width>35</width>
                     <height>35</height>
                    </size>
                   </property>
                   <property name="text">
                    <string/>
                   </property>
                   <property name="pixmap">
                    <pixmap>../img/destop.png</pixmap>
                   </property>
                   <property name="scaledContents">
                    <bool>true</bool>
                   </property>
                  </widget>
                 </widget>
                </item>
                <item>
                 <widget class="QWidget" name="widget_11" native="true">
                  <property name="styleSheet">
                   <string notr="true">background-color: #e2e2a9;
border-radius: 5px</string>
                  </property>
                  <widget class="QLabel" name="label_12">
                   <property name="geometry">
                    <rect>
                     <x>7</x>
                     <y>5</y>
                     <width>35</width>
                     <height>35</height>
                    </rect>
                   </property>
                   <property name="maximumSize">
                    <size>
                     <width>35</width>
                     <height>35</height>
                    </size>
                   </property>
                   <property name="text">
                    <string/>
                   </property>
                   <property name="pixmap">
                    <pixmap>../img/sign-out-option.png</pixmap>
                   </property>
                   <property name="scaledContents">
                    <bool>true</bool>
                   </property>
                  </widget>
                 </widget>
                </item>
                <item>
                 <widget class="QWidget" name="widget_12" native="true">
                  <property name="styleSheet">
                   <string notr="true">background-color: #e2e2a9;
border-radius: 5px</string>
                  </property>
                  <widget class="QLabel" name="label_13">
                   <property name="geometry">
                    <rect>
                     <x>7</x>
                     <y>5</y>
                     <width>35</width>
                     <height>35</height>
                    </rect>
                   </property>
                   <property name="maximumSize">
                    <size>
                     <width>35</width>
                     <height>35</height>
                    </size>
                   </property>
                   <property name="text">
                    <string/>
                   </property>
                   <property name="pixmap">
                    <pixmap>C:/Users/DO MANH HUNG/Downloads/sign-out-option.png</pixmap>
                   </property>
                   <property name="scaledContents">
                    <bool>true</bool>
                   </property>
                  </widget>
                 </widget>
                </item>
               </layout>
              </widget>
             </widget>
             <widget class="QLabel" name="label">
              <property name="geometry">
               <rect>
                <x>110</x>
                <y>10</y>
                <width>621</width>
                <height>51</height>
               </rect>
              </property>
              <property name="font">
               <font>
                <pointsize>22</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Welcome to food manager</string>
              </property>
             </widget>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
